src/goPath.ts

```diff
@@ -27,6 +28,13 @@ export function getBinPath(binname) {
 	return path.join(process.env["GOPATH"], "bin", binname);
```

Comment:
`workspaces` will always be of `.length > 0` and if no bin is found returning `ws1:ws2/bin/name` makes no sense.